ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -615,6 +615,40 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-mce-arm-nodepool-mgmt-f7
+  cron: 35 14 5,12,19,28 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      RELEASE_IMAGE_LATEST: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRA_ARGS: --arch=arm64
+      HYPERSHIFT_INSTANCE_TYPE: m6g.xlarge
+      MCE_VERSION: "2.7"
```

Comment:
I can see some 4.17 jobs are using MCE 2.6. Could you explain a little bit that how to decide we should use 2.6 or 2.7 ?

Reply:
Based on the current rules, we should all be using version 2.7. Currently, e2e is using 2.7, while upgrade is using 2.6. I will submit a new PR to update all the upgrade CI to version 2.7.